src/ipc/reboot.ts

```diff
@@ -7,7 +7,7 @@ export const channel = 'reboot';
  * Reboot the machine
  */
 export default function register(ipcMain: IpcMain): void {
-  ipcMain.handle(channel, () => {
-    exec('reboot');
+  ipcMain.handle(channel, async () => {
+    await exec('reboot');
```

Comment:
we should probably do the same thing here as we do for reboot from BIOS. Either `void` both, or `await` both.

Reply:
I switched to `void` because intuitively it made more sense to me to just trigger a reboot rather than wait on it, because the machine is powering down.